dev/com.ibm.ws.beanvalidation.v20_fat/fat/src/com/ibm/ws/bval/v20/fat/BeanVal20Test.java

```diff
@@ -54,43 +40,52 @@
     @Server("beanval.v20_fat")
     @TestServlets({
                     @TestServlet(servlet = BeanVal20TestServlet.class, contextRoot = REG_APP),
-                    @TestServlet(servlet = BeanValCDIServlet.class, contextRoot = CDI_APP),
-                    @TestServlet(servlet = BeanValidationTestServlet.class, contextRoot = MULTI_VAL_APP),
-                    @TestServlet(servlet = ValueExtractorServlet.class, contextRoot = VAL_EXT_APP),
+                    //TODO: Once CDI 3.0 and EJBLite 4.0 are ready, enable these four applications for Jakarta EE 9.
+//                    @TestServlet(servlet = BeanValCDIServlet.class, contextRoot = CDI_APP),
```

Comment:
We don't want to disable these tests outright because we will lose coverage for bval-2.0 in the short term.
Instead, can we just disable these tests using `@SkipForRepeat` on the test servlet class? Currently you can use `@SkipForRepeat` on test servlet methods, but not at the class level. It would be nice to have class-level support, but if that's too much work you could just add the annotation on each test servlet method instead

Reply:
The tests are temporarily disabled here. They have been moved to their own test class that only runs for bval 1.1 and bval 2.0, so we don't lose coverage.